message_ix_models/model/material/data_util.py

```diff
@@ -2223,3 +2220,35 @@ def add_elec_i_ini_act(scenario: message_ix.Scenario) -> None:
     scenario.add_par(par, df_el)
     scenario.commit("add initial_activity_up for elec_i")
     return
+
+
+def calculate_ini_new_cap(
+    df_demand: pd.DataFrame, technology: str, material: str
+) -> pd.DataFrame:
+    """
+    Derive initial_new_capacity_up parametrization for CCS based on cement demand
+    projection
+    Parameters
+    ----------
+    df_demand: pd.DataFrame
+        DataFrame containing "demand" MESSAGEix parametrization
+    technology: str
+        name of CCS technology to be parametrized
+    material: str
+        name of the material/industry sector
+    Returns
+    -------
+    DataFrame formatted to "initial_new_capacity_up" columns
+    """
+
+    SCALER = 0.005
+
+    if material == "cement":
+        CLINKER_RATIO = 0.72
+        df_demand["value"] *= CLINKER_RATIO * SCALER
+    else:
+        df_demand["value"] *= SCALER
```

Comment:
These lines require two tests at the moment: one where `material` is `"cement"` and one where it is another `str`. We can reduce the number of required tests (and perceived complexity, maybe) like so:
```suggestion
    CLINKER_RATIO = 0.72 if material == "cement" else 1
    df_demand["value"] *= CLINKER_RATIO * SCALER
```